debian-snapper-nodejs/run_node

```diff
@@ -9,5 +8,10 @@ $(test "$(ls -A "$NODE_APP_DIR" 2>/dev/null)") || \
 echo "==> Installing npm dependencies"
 npm install
 
-echo "==> Starting the server"
-exec npm start
+if [ "$RUN_NODEMON" = "true" ]; then
```

Comment:
perhaps this is assigning the value to true instead of acting as a conditional? is single-equals similar to PHP/JS or does it behave differently in this context?

Reply:
Nah, this is correct bash (sh) syntax.